### PR TITLE
[Design System] Create a general README.md (Documentation) for componentsLibrary(DSCO)

### PR DESCRIPTION
**Pull request by levadadenys**
### [Design System] Create a general README.md (Documentation) for componentsLibrary(DSCO)
* create a general Readme.md for componentLibrary (Design system)
## Links
- jira ticket: [DES-312](https://codedotorg.atlassian.net/browse/DES-312)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/componentLibrary/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+=============================================
+This is a collection of Design System React components that are used across our apps.
+We store components in `src/componentLibrary` folder(the one where this README.md is located) for now,
+but once we'll be ready - we'll be moving design system components to
+[DSCO_ repo](https://github.com/code-dot-org/dsco_).
+
+The main idea of Design system is that you'll use these components when you'll need to create a new page or update
+an existing one, and you'll be able to do it without a designer, because you'll have all the components you need
+in one place. In addition to that, we'll have one instance of base components meaning that once we'll want to update
+something across the project(e.g. change color scheme of button, or Typography styles), we'll only need to update this
```
Comment (sanchitmalhotra126) [inline apps/src/componentLibrary/README.md]: super nit: space between "project" and parentheses ("project (e.g. change ...")

File: apps/src/componentLibrary/README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+Components Library (Design System components)
```
Comment (sanchitmalhotra126) [inline apps/src/componentLibrary/README.md]: nit: *Component Library (for consistency 🙂)

**Review by sanchitmalhotra126 — APPROVED** [review]
This is great! Thanks for the thorough documentation!

As a follow-up, it would also be good to add some documentation for other developers who want to contribute/work on components - best practices, paradigms to use (i.e. functional components, typescript, etc), testing libraries to use, documentation to include (ex. CHANGELOG and README), any style recommendations, etc

**Comment by levadadenys** [comment]
> As a follow-up, it would also be good to add some documentation for other developers who want to contribute/work on components - best practices, paradigms to use (i.e. functional components, typescript, etc), testing libraries to use, documentation to include (ex. CHANGELOG and README), any style recommendations, etc

@sanchitmalhotra126 
oh yeah, that's a great idea thanks! There was an idea of creating some best practices, patterns and hints for the whole repo/developing new components, but it was never started because of a lot of work on design system. I guess it might be even better to write those hints for components library(less preassure and probably faster time of updating)

**Pull request merged**